azure/template.json

```diff
@@ -359,7 +447,9 @@
                 }
             },
             "dependsOn": [
-                "function-app-service-plan"
+                "function-app-service-plan",
+                "[concat(variables('importFunctionAppName'),'-insights')]",
```

Comment:
Implicit dependency created for app-insights and subnet by use of reference function